.github/workflows/e2e.yml

```diff
@@ -13,13 +13,20 @@ on:
 jobs:
   cypress-run:
     runs-on: ubuntu-20.04
+    strategy:
```

Comment:
Changes on this `yml` were necessary to make the CI pass, but are unrelated to the scope of this PR.

Reply:
Dropped this commit, since it has no relation to this PR.
This workflow issue that still persists will be solved on #393 and merged back here as soon as it's available in `dev`

Reply:
Merged and the CI is running again!